configserver/src/main/java/com/yahoo/vespa/config/server/tenant/TenantRepository.java

```diff
@@ -300,11 +300,7 @@ public Instant readCreatedTimeFromZooKeeper(TenantName tenantName) {
 
     // Creates tenant and all its dependencies. This also includes loading active applications
     private Tenant createTenant(TenantName tenantName, Instant created) {
-        if (tenants.containsKey(tenantName)) {
-            Tenant tenant = getTenant(tenantName);
-            createAndWriteTenantMetaData(tenant);
-            return tenant;
-        }
+        if (tenants.containsKey(tenantName)) return getTenant(tenantName);
```

Comment:
this change must be a relic of the other PR.

Reply:
ah, as you say "includes changes in #16196"

Reply:
yup